src/Rhythm/Revit/Application/Application.cs

```diff
@@ -106,5 +106,25 @@ public static List<Autodesk.Revit.DB.Document> GetOpenDocuments(bool runIt)
             }
             return documents;
         }
+
+        /// <summary>
+        /// This node will save the Revit document to another path.
+        /// </summary>
+        /// <param name="Document">A valid Revit Document.</param>
+        /// <param name="FilePath">The file path to save the document.</param>
+        /// <returns name="result">A string message whether the save as was successful or a failure.</returns>
+        [NodeCategory("Action")]
+        public static string SaveAsDocument(Revit.Application.Document Document, string FilePath)
```

Comment:
rename the method to just SaveAs. This would allow it to show in Dynamo as `Documents.SaveAs`